components/mpas-ocean/src/shared/mpas_ocn_surface_bulk_forcing.F

```diff
@@ -346,6 +424,7 @@ subroutine ocn_surface_bulk_forcing_vel( &
       deallocate(zonalWSEdge, meridWSEdge)
 
       !*** Transfer any needed data back to host and delete inputs
+      !$acc copyout(ekeCorrection)
```

Comment:
move these three data lines up inside the MPAS OPENACC ifdef just above with the other data delete clause. Also add missing clause (assuming delete) for the remaining arrays that were copied in: windSpeed10m, iceFraction, velocityZonal, velocityMeridional, normalVelocity (I don't remember whether we have those permanently on device?  If so, you don't need the copyin or delete)

Reply:
That's all I saw looking through.